### DISCUSSION TITLE
Mass price recalculation

### DISCUSSION THREAD

**Discussion by einhverfr** [ledgersmb/LedgerSMB#5835] [Ideas]
via  @pongraczi 
Mass recalculation also would be nice:
imagine an imported product list, where the purchasing price in USD, but selling price is HUF (EUR, whatever) -> as the exchange rate is changing, it would be nice to adjust the price.
Example: adjusting by percent or fixed amount.
Original price: 100
Increasing it by 10% -> new price: 110
Increasing it by adding 20 -> new price: 120

**Comment by ehuelsmann** [comment]
Instead of the proposed solution, a "price update upload" would be a very flexible solution: the price update could be done in a spreadsheet while the complexity of LedgerSMB stays low.
